pkg/trait/container.go

```diff
@@ -28,6 +28,11 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	defaultContainerName = "integration-container"
```

Comment:
I would suggest `integration` container rather than the `integration-container` container.